Makefile

```diff
@@ -135,10 +135,16 @@ container-tests: pkg/apis/wksprovider/machine/scripts/scripts_vfsdata.go pkg/api
 
 integration-tests-container: cmd/wksctl/wksctl
 	APK_INDEX='https://https:dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz https://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz'
+
+# Setting environment variable NODE_OS will run the test with the specified OS
+# Valid values are centos or ubuntu, e.g NODE_OS=ubuntu make integration-tests-container
+
+# Leaving NODE_OS unset will result in one of centos or ubuntu being chosen for the test
+
 	NODE_OS1="centos"
 	NODE_OS2="ubuntu"
-	NODE_OS_CHOICE=$${NODE_OS_CHOICE:-"$$(echo $$NODE_OS1 $$NODE_OS2 | tr ' ' '\n' | shuf | head -1)"}
-	IMAGE_TAG=$(IMAGE_TAG) NODE_OS="$${NODE_OS_CHOICE}" go test -v -timeout 40m ./test/integration/container/...
+	NODE_OS_CHOICE=$${NODE_OS:-"$$(echo $$NODE_OS1 $$NODE_OS2 | tr ' ' '\n' | shuf | head -1)"}
+	IMAGE_TAG=$(IMAGE_TAG) NODE_OS="$${NODE_OS}" go test -v -timeout 40m ./test/integration/container/...
```

Comment:
I believe the second use of NODE_OS here should be NODE_OS_CHOICE

Reply:
Have replaced NODE_OS_CHOICE with just NODE_OS